cmds/replay.c

```diff
@@ -985,8 +985,6 @@ static int print_graph_rstack(struct uftrace_data *handle,
 			if (opts->comment) {
 				pr_gray(" /* %s%s%s ", symname,
 					*libname ? "@" : "", libname);
-				if (str_loc)
-					pr_gray("at %s ", str_loc);
 				pr_gray("*/");
```

Comment:
You can merge this into the above `pr_gray`.

Reply:
It's fixed. Thank you for fine comment.